featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -1120,7 +1120,7 @@ def test_text_primitives(es):
         sort_index=True,
     )
 
-    word_counts = [514, 3, 3, 644, 1268, 1269, 177, 172, 79, 240, 1239, 3, 3, 3, 3]
+    word_counts = [532, 3, 3, 653, 1306, 1305, 174, 173, 79, 246, 1253, 3, 3, 3, 3]
```

Comment:
One final question. In the original issue it says that adding whitespace characters with the old approach would cause counts to increase, which was not accurate. In this test, we are seeing increases in counts. Do we understand why this is and are these correct? Do we expect word counts to increase with this change?

Reply:
I believe this to be a consequence of how we were handling tabs and newlines. In the old way, we counted the whitespace " " to infer the number of words. However, this would lead to the string ```soft drinks:\n\nDiet Coke``` (taken from the entityset) being inferred as 3 words. Our current implementation splits the string, and would count the above as having 4 words.

Reply:
Got it. So now because we also split on tabs and new line characters we end up with more words, even though we now properly handle multiple spaces in a row.

Reply:
Yep, to summarize for future reference, these changes 
(1) decrease our count because we stop recognizing words like "hello   world" as having 4 words simply because it has 3 spaces
(2) adds to our count because we now split on tabs and newlines, which we weren't considering previously 
I think this explains why some of our tests have increased number of words, and some have decreased number of words, and this seems to hold up under manual inspection.